open-machine-learning-jupyter-book/assignments/ml-fundamentals/web-app.md

```diff
@@ -0,0 +1,11 @@
+# Try a different model
+
+## Instructions
+
+Now that you have built one web app using a trained Regression model, use one of the models from an earlier Regression lesson to redo this web app. You can keep the style or design it differently to reflect the pumpkin data. Be careful to change the inputs to reflect your model's training method.
+
+## Rubric
```

Comment:
You need to add the acknowledge section. E.g. https://open-academy.github.io/machine-learning/assignments/data-science/small-diabetes-study.html#acknowledgments